recipes/eodatasets3/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "eodatasets3" %}
+{% set version = "0.29.4+1.g61dddc9" %}
```

Comment:
Is this an actual release or a development package? The version number is uncommon for a release.

Reply:
fixed, please re-review